### ISSUE TITLE
Actual SourceMap default (Docu)?

### ISSUE THREAD

**Issue by pablopaul** [JeffreyWay/laravel-mix#1134]
Hey @JeffreyWay and crew, 

whats the actual `.sourceMaps()` default output? Is:

* in dev build: inlined source maps
* in prod build: source maps in separated file

correct?

Where is that documented right now, having a hard time to find info about it 😄 

Thanks,
Paul

**Comment by JeffreyWay** [comment]
The default webpack source map plugins are:

Local: cheap-module-eval-source-map
Production: cheap-source-map

**Comment by pablopaul** [comment]
@JeffreyWay thanks for your answer, what would be the best place to document that?

**Comment by dotnetCarpenter** [comment]
I've had an issue with source maps not showing the correct line for months and while it's been a pain, I've lived with it until today. I wish I had done this so much sooner and I hope this will help others.

After doing a source code search I found that [sourceMaps takes a second argument, `type`](https://github.com/JeffreyWay/laravel-mix/blob/master/src/Api.js#L12) which correspond to [the source map devtool list on the webpack website](https://webpack.js.org/configuration/devtool/#devtool). Settings this to anything else than the default `eval-source-map`, worked great!

PS. Not finding this written anywhere on https://laravel-mix.com/docs/2.1/
